tests/local/network.go

```diff
@@ -55,7 +55,14 @@ type LocalNetwork struct {
 }
 
 const (
-	fundedKeyStr = "56289e99c94b6912bfc12adc093c9b51124f0dc54ac7a766b2bc5ccf558d8027"
+	fundedKeyStr           = "56289e99c94b6912bfc12adc093c9b51124f0dc54ac7a766b2bc5ccf558d8027"
+	warpEnabledChainConfig = `{
+		"warp-api-enabled": true, 
+		"eth-apis":["eth","eth-filter","net","admin","web3",
```

Comment:
what is the `eth-apis` config field used for, is it required?

Reply:
They're different collections of API endpoints that the node can chose to run or not. There's a default set, and then some more advanced ones that not everyone might want to expose. `TraceTransaction` uses one of the non-default APIs, and I've just enabled all the APIs I could find here to make it easier to use others in the future.